substrate/client/transaction-pool/src/fork_aware_txpool/multi_view_listener.rs

```diff
@@ -445,16 +458,15 @@ where
 	///
 	/// This method sends a `TransactionDropped` command to the controller of each requested
 	/// transaction prompting and external `Broadcasted` event.
-	pub(crate) fn transactions_dropped(&self, dropped: &[ExtrinsicHash<ChainApi>]) {
+	pub(crate) fn transaction_dropped(&self, dropped: DroppedTransaction<ExtrinsicHash<ChainApi>>) {
 		let mut controllers = self.controllers.write();
-		debug!(target: LOG_TARGET, "mvl::transactions_dropped: {:?}", dropped);
-		for tx_hash in dropped {
-			if let Some(tx) = controllers.remove(&tx_hash) {
-				debug!(target: LOG_TARGET, "[{:?}] transaction_dropped", tx_hash);
-				if let Err(e) = tx.unbounded_send(ControllerCommand::TransactionDropped) {
-					trace!(target: LOG_TARGET, "[{:?}] transactions_dropped: send message failed: {:?}", tx_hash, e);
-				};
-			}
+		debug!(target: LOG_TARGET, "mvl::transaction_dropped: {:?}", dropped);
```

Comment:
nit: Maybe we can introduce a new `LOG_TARGET` for the mvl similar to `LOG_TARGET: &'static = "txpool::mvl"`?

Reply:
I don't think `mvl` is not standalone target which could be enabled on its own. Not sure if I want to mess with many targets for txpool. Maybe at some point in future.

Reply:
Second thought - maybe it makes sense only for having the pattern.
I'll be reworking logs in future, will take this into account. (https://github.com/paritytech/polkadot-sdk/issues/5490)